src/SparkTestBase.sol

```diff
@@ -191,6 +225,107 @@ abstract contract SparkTestBase is ProtocolV3TestBase {
         }
     }
 
+    function testRewardsConfiguration() public {
+        uint256 snapshot = vm.snapshot();
+
+        _testRewardsConfiguration();
+
+        vm.revertTo(snapshot);
+        GovHelpers.executePayload(vm, payload, executor);
+
+        _testRewardsConfiguration();
+    }
+
+    function _testRewardsConfiguration() public {
```

Comment:
Would use a different name here. `_assertRewardsConfiguration` is clearer it's not a test itself. Also should be `internal` visibility.